.gitmodules

```diff
@@ -20,7 +20,7 @@
 [submodule "nitro-testnode"]
 	path = nitro-testnode
 	url = https://github.com/EspressoSystems/nitro-testnode
-        branch = integration
+	branch = jh/deploy
```

Comment:
Change this back to integration?

Reply:
Yeah after this branch is merged to the integration.